Packs/EDLMonitor/Integrations/EDLMonitor/EDLMonitor.py

```diff
@@ -0,0 +1,268 @@
+
+
+def sendemail(reason, create_ts, email, email_server, email_user, email_pwd, txtfile=None):
+
+    if txtfile != None:
+        zf = tempfile.TemporaryFile(prefix='EDL monitor results', suffix='.zip')
+        zip = zipfile.ZipFile(zf, 'a')
+        zip.write(txtfile)
+        zip.close()
+        zf.seek(0)
+
+        # Create the message
+        themsg = MIMEMultipart()
+        if reason == "test":
+            themsg['Subject'] = f'Test success, EDL contents at {create_ts}! Instance- {instance_name}'
+        else:
+            themsg['Subject'] = 'EDL monitor and Checker results for %s' % instance_name
+        themsg['To'] = email
+        sender = "noreply@demisto.com"
+        themsg['From'] = sender
+        themsg.preamble = 'I am not using a MIME-aware mail reader.\n'
+        msg = MIMEBase('application', 'zip')
+        msg.set_payload(zf.read())
+        encoders.encode_base64(msg)
+        msg.add_header('Content-Disposition', 'attachment',
+                       filename="EDL Monitor Results.zip")
+        themsg.attach(msg)
+    else:
+        # An availability email only
+
+        # Email current EDL to user
+        sender = "noreply@demisto.com"
+        receivers = [email]
+        demisto.debug(f'EDL monitor availability email send to - ' + email)
+
+        # Create the message
+        themsg = MIMEMultipart()
+        if reason == "timeout":
+            themsg['Subject'] = f'EDL timeout at {create_ts}! Instance- {instance_name}'
+        else:
+            themsg['Subject'] = f'EDL down at {create_ts}! Instance- {instance_name}'
+        themsg['To'] = email
+        sender = "noreply@demisto.com"
+        themsg['From'] = sender
+
+        #part1 = MIMEText(text, 'plain')
+
+        # themsg = f"""Subject: EDLmonitor content for {EDL}
+        # EDL contents as of {pull_time}:
+        # {linebreak.join(results)}
+        # """
+    themsg_str = themsg.as_string()
+    try:
+        # Enable additional smtplib logs
+        #smtplib.SMTP.debuglevel = 1
+        SERVER = smtplib.SMTP(email_server, 587)
+        SERVER.ehlo()  # type: ignore
+        # if TLS is True or TLS == 'STARTTLS' or str(TLS).lower() == 'true':
+        SERVER.starttls()  # type: ignore
+        if email_user != "":
+            demisto.debug('EDL monitor Email AUTHING')
+            # demisto.debug(f'EDL monitor Email - ' + email_user + " " + email_pwd)  ## CAREFUL with this line for debugging only!!!!
+            SERVER.login(email_user, email_pwd)
+
+        SERVER.sendmail(sender, email, themsg_str)
+        # SERVER.sendmail(FROM, to + cc + bcc, str_msg)  # type: ignore[union-attr]
+        SERVER.quit()
+        demisto.debug('EDL monitor Successfully sent email')
+    except Exception as err:
+        raise ValueError("Failed to send email: " + str(err))
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(EDL, edl_user, edl_pwd, verify_certificate, pull_time, email, email_server, email_user, email_pwd, timeout) -> str:
```

Comment:
EDL, edl_user, edl_pwd, verify_certificate, pull_time, timeout are not in use in this function? why are they being passed to the function?